### DISCUSSION TITLE
Suggestion for improvement. (Please introduce a driver for Windows)

### DISCUSSION THREAD

**Discussion by erashmika** [NaitLee/Cat-Printer#84] [Ideas]
@NaitLee 
Can you create a **DRIVER FOR WINDOWS** that can work well with these mini printers?

Then we can get any print easily without using any 3rd party software. Like a normal printer. (Plug And Play)
Also, we can connect the mini printer to a POS software and print something like a receipt very easily.

I hope you will give a favorable response to my proposal soon. Also, I hope that the driver I mentioned will be introduced in your next update. I appreciate your hard work for this. Thank you very much. ❤️❤️❤️

**Comment by ricdk** [comment]
yes a driver would be really nice.

**Comment by ValdikSS** [comment]
There's IPP support, you can add this printer as an IPP printer.
